ppr-ui/tests/unit/EffectiveDateTime.spec.ts

```diff
@@ -0,0 +1,47 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
+import { getTestId } from './utils'
+import { EffectiveDateTime } from '@/components/unitNotes'
+import { SharedDatePicker } from '@/components/common'
+
+Vue.use(Vuetify)
+const vuetify = new Vuetify({})
+
+/**
+ * Creates and mounts a component, so that it can be tested.
+ *
+ * @returns a Wrapper<Any> object with the given parameters.
+ */
+function createComponent (propsData: any): Wrapper<any> {
+  const localVue = createLocalVue()
+  localVue.use(Vuetify)
+
+  return mount((EffectiveDateTime as any), {
+    localVue,
+    propsData,
+    vuetify
+  })
+}
```

Comment:
I believe there is a common function from the test utils folder we can use. 
Trying to get away from having this function in every test file.

Reply:
Updated.